cmd/ci-operator-prowgen/testdata/zz_fixture_presubmit_TestFromCIOperatorConfigToProwYaml_one_test_and_images__no_previous_jobs._Expect_test_presubmit__pre_post_submit_images_jobs.yaml

```diff
@@ -58,7 +58,7 @@ presubmits:
     decoration_config:
       skip_cloning: true
     labels:
-      ci-operator.openshift.io/prowgen-controlled: newly-generated
+      ci-operator.openshift.io/prowgen-controlled: "true"
```

Comment:
I'm surprised at this - why? Does this impact pruning?

Reply:
This is because writing + pruning are a single step now (see below).  There are plenty of pruning scenarios in the integration tests, I imagine these predate them.